src/internal/utils/file_utils.go

```
@@ -37,66 +37,59 @@
-		if err = toml.Unmarshal(data, target); err != nil {
-			hasError = true
-			//nolint: errorlint // Type assertion is better here, and we need to read data from error
-			if decodeErr, ok := err.(*toml.DecodeError); ok {
-				row, col := decodeErr.Position()
-				errMsg = errorPrefix + fmt.Sprintf("Error in field at line %d column %d: %s\n",
-					row, col, decodeErr.Error())
-			} else {
-				errMsg = errorPrefix + "Error unmarshalling data: " + err.Error() + "\n"
-			}
+	if err := toml.Unmarshal(data, target); err != nil {
+		if decodeErr, ok := err.(*toml.DecodeError); ok {
```

golangci-lint (1.64.8)

type assertion on error will fail on wrapped errors. Use errors.As to check for specific errors

(errorlint)